examples/reverse-proxy/MNIST_using_Reverse_Proxy.md

```diff
@@ -0,0 +1,70 @@
+
+The following image illustrates a cluster setup for the Reverse Proxy with MNIST example:
+
+![](/docs/User/GUID-D9C8F73B-BAFB-41E2-9B70-02C628836B96-high.png)
+
+1.  This example uses a SN node. This node is named as **sn-1** and is the Sentinel Node. **sn-1** runs on host 172.1.1.1.
+
+2.  SL and ML nodes are automatically spawned by SWOP node during training.
+
+3.  This example uses a SWOP node that connects to SN node. The name of the docker container representing this SWOP node is **swop-1**. **swop-1** runs on host 172.1.1.1.
+
+4.  Training is initiated by SWCI node \(**swci-1**\) that runs on host 172.1.1.1.
+
+5.  This example assumes that the License Server already runs on host 172.1.1.1. All Swarm nodes connect to the License Server, on its default port 5814.
+
+
+This example runs Reverse Proxy with MNIST [1] on the Swarm Learning platform. It uses TensorFlow as the backend. The code for this example is taken from [2] and modified to run on a Swarm Learning platform.
+
+This example shows the MNIST using reverse proxy to mimic real world behaviour. This example uses BIND9 as the DNS server and NGINX as the reverse proxy server and builds both the docker images with suitable configurations. \(Please refer to the respective docker files\). This example has automated the end-to-end running of MNIST, and includes starting the BIND9 and NGINX containers for user convenience. For more information on arguments passed to the respective run scripts of swarm components, see `run-all` script.
+
+This example uses one training batch and one test batch. Both batch files are stored in an archive file called `mnist.npz`. The Machine Learning program, after conversion to Swarm Learning for the TensorFlow-based Keras platform, is in `examples/reverse-proxy/mnist/model` and the TensorFlow-based file name is `mnist tf.py`.
+
+This example shows the Swarm training of MNIST model using two ML nodes. ML nodes are automatically spawned by SWOP node. Swarm training is initiated by SWCI node and orchestrated by a SN node. This example also shows how private data and shared model can be mounted to ML nodes for Swarm training. For more information, see the profile files and task definition files placed under `examples/reverse-proxy/mnist/swop` and `examples/reverse-proxy/mnist/swci` folders, respectively.
+
+## Running the MNIST example
+
+1.  On host-1, navigate to swarm-learning folder.
+
+    ``` {#CODEBLOCK_MYT_33W_NVB}
+    cd swarm-learning
+    ```
+
+2.  Run the `run-all` script from the `swarm-learning` folder with `APLS IP` argument and `wait time for sn`.
+
+    ``` {#CODEBLOCK_LQT_DNY_NVB}
+    .\examples\reverse-proxy\run-all 172.1.1.1 4m
+    ```
+
+    First argument `APLS IP` in this command is a mandatory parameter. Second argument `wait time for sn` if not passed, by default it will wait for 5 minutes. This `wait time for sn` argument is similar to sleep, where `m` denotes minutes.
```

Comment:
where `m` denotes minutes -> where `m` denotes minutes.  This script waits for SN node to initialize fully within `m` minutes, before proceeding with the next steps.